src/components/tree-views-container.tsx

```diff
@@ -129,11 +146,11 @@ function updatedTree(prevRoots, prevMap, nodeId, children) {
         ...Array.prototype.concat(
             ...prevUuids
                 .filter((u) => !nextUuids.has(u))
-                .map((u) => flattenDownNodes(prevMap[u], (n) => n.children).map((n) => n.elementUuid))
+                .map((u) => flattenDownNodes(prevMap[u], (n: IDirectory) => n.children).map((n) => n.elementUuid))
         ),
     ]);
 
-    const prevNode = nodeId ? prevMap[nodeId] : {};
+    const prevNode = nodeId && prevMap ? prevMap[nodeId] : {};
```

Comment:
```suggestion
    const prevNode = nodeId && prevMap?[nodeId] : {};
```

Reply:
Don't understand this suggested change ... it's syntactically incorrect, no ?

Reply:
This changes the behavior, yoiu can stick with previous syntax